Worksheet_C/README.md

```diff
@@ -0,0 +1,11 @@
+# Worksheet C
+
+- (a) This algorithm scans through a single list and compares every single item to check for duplicate items. 
+- (b) The reason that this algorithm is quadratic, is because everytime you add a new item the algorithm has to run the new addition against every possible result in the list. 
+- (c) The algorithm is still correct, as it runs previous items in the list against the new item meaning that once completed it will have compared all items but doesn't make duplicate comparisons. 
+- (d) The algorithm is half as fast because it only has to make half the comparisons after removing the duplicate comparisons. 
+- (e) Yes it is still quadratic as it does n “things” and each “thing” is O(n), then the overall algorithm is O(n**2). 
+- (f) http://stackoverflow.com/questions/23809785/python-sorting-complexity-on-sorted-list - "Best case scenario: O(n), average and worst case O(nlogn). – Martijn Pieters". 
+- (g) I think the answer would be O(nlogn) as that is the worst case scenario of the sorting function. 
```

Comment:
Yes, but why is it the sorting function that dominates and not the for loop in lines 4-8?